src/target/riscv/riscv.h

```diff
@@ -60,6 +60,12 @@ typedef struct {
 	unsigned custom_number;
 } riscv_reg_info_t;
 
+typedef struct {
+	struct list_head list;
+	uint16_t low, high;
+	char *id;
```

Comment:
I think "name" is more descriptive than "id".

Reply:
Changed ids to names